tests/sqlplannertest/src/lib.rs

```diff
@@ -13,7 +12,7 @@ pub struct DatabaseWrapper;
 impl sqlplannertest::PlannerTestRunner for DatabaseWrapper {
     async fn run(&mut self, test_case: &ParsedTestCase) -> Result<String, Error> {
         if !test_case.tasks.is_empty() {
-            let db = Database::new_on_disk(SecondaryStorageOptions::default_for_test()).await;
+            let db = Database::new_in_memory();
```

Comment:
In-memory engine might not support predicate pushdown on table scan. I still want to keep new_on_disk with in-memory backend here.